### DISCUSSION TITLE
standalone SO with No Alert and Grid status shown Fault

### DISCUSSION THREAD

**Discussion by lucc4amba** [Security-Onion-Solutions/securityonion#12930] [Unsupported Versions]
New on SO. grid status shown "Fault". 
so-status shown OK
any suggest to debug?

**Comment by Alex-Chau** [comment]
I have the same issue with the sensor test and no Alert.

**Reply by dougburks** [reply]
@Alex-Chau Please try the steps in the Troubleshooting Alerts section of the documentation:
https://docs.securityonion.net/en/2.4/suricata.html#troubleshooting-alerts

If that doesn't help, then please start a new discussion (instead of replying to this discussion) and provide the results of each of the steps and more information about your system.

**Comment by dougburks** [comment]
@lucc4amba Please try the steps in the Troubleshooting Alerts section of the documentation:
https://docs.securityonion.net/en/2.4/suricata.html#troubleshooting-alerts

If that doesn't help, you'll need to provide the results of each of the steps and more information about your system.
